golden_response/staging/get_landing_page/county.json

```diff
@@ -267,6 +267,23 @@
               "2015": 1918044
             }
           },
+          {
+            "metadata": {
+              "importName": "CensusACS5YearSurvey",
+              "measurementMethod": "CensusACS5yrSurvey",
+              "domain": "census.gov"
+            },
+            "timeSeries": {
+              "2011": 1762754,
+              "2012": 1788393,
+              "2013": 1812208,
+              "2014": 1841569,
+              "2015": 1868149,
+              "2016": 1885056,
+              "2017": 1911226,
+              "2018": 1922200
+            }
+          },
```

Comment:
@shifucun I thought we had done some ordering and serialization to make sure these ordering diffs don't show?